installer.nsh

```diff
@@ -18,9 +18,16 @@
     ; stop the agent
     nsExec::ExecToStack /OEM 'powershell "& " "$\'"$path_i\remoteit.exe$\'" -j agent stop'
 
-    ; move the config file and connections to backup location (protect against 2.9.2 uninstall bug)
-    Rename "$APPDATA\remoteit\config.json" "${REMOTEIT_BACKUP}\config.json"
```

Comment:
First we delete ${REMOTEIT_BACKUP}\config.json.2.9.5 if it exists
then we can safely move
Rename "$APPDATA\remoteit\config.json" "${REMOTEIT_BACKUP}\config.json.2.9.5"

Reply:
Would be nice to automate this file name interpolation with a $version or something

Reply:
if the file exists it is likely a good backup and we should not delete it or replace with anything

Reply:
Maybe we should delete it first... so if they re-install again after setting stuff up / changing thing it won't restore an old file. Or we should move it back and not leave it in backup... make it just a temp file.